test/src/test/java/hudson/diagnosis/OldDataMonitorTest.java

```diff
@@ -152,6 +154,17 @@ public Void call() {
 
     }
 
+    @Issue("JENKINS-63372")
+    @LocalData("main")
```

Comment:
There is no resource for this. Forgot to commit?

Reply:
> There is no resource for this. Forgot to commit?

Looking at the test - `LocalData` seems to be uneeded?

```suggestion
```